db/schema.rb

```diff
@@ -465,6 +465,12 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.string   "picture"
+    t.boolean  "payed",                default: false
+    t.boolean  "swags",                default: false
+    t.boolean  "swags_received"
```

Comment:
If we go with `has_swag`, let's have that be `swag_received`, also in singular. Again this is for uniformity in names, making things simple and easy to remember and use.